src/components/proofs/List.tsx

```diff
@@ -1,11 +1,24 @@
+import { Suspense } from 'preact/compat'
 import ListTitle from 'components/proofs/ListTitle'
+import LoadingCard from 'components/proofs/LoadingCard'
 import ProofList from 'components/proofs/ProofList'
+import useProofAddressesAvailableToCreate from 'hooks/useProofAddressesAvailableToCreate'
+
+function ListSuspended() {
+  const avaliableToProof = useProofAddressesAvailableToCreate()
```

Comment:
Let's rename it to `avaliableToProofList`, because it doesn't explain what is available to proof